eppy/modeleditor.py

```diff
@@ -551,6 +551,12 @@ def __init__(self, idfname=None, epw=None):
         if epw != None:
             self.epw = epw
         self.outputtype = "standard"
+    
+    def __enter__(self):
+        return self
+
+    def __exit__(self, type, value, traceback):
+        pass
```

Comment:
Is there any cleanup that should happen here? Having these two methods means that `IDF` can be used in a context manager:

    with IDF("myidf.idf", "myepw.epw") as myidf:
        idf.run()
        # idf.__exit__() could handle tidying up temporary files, etc in case EnergyPlus crashes

Reply:
Good idea. I wouldn't know how to implement that 😒